src/phylum/ci/ci_base.py

```diff
@@ -119,8 +128,8 @@ def check_prerequisites(self) -> None:
             #       https://github.com/phylum-dev/phylum-ci/issues/31
             raise SystemExit(" [!] The `.phylum_project` file was not found at the current working directory")
 
-        if Version(self.args.version) < Version("v3.3.0-rc1"):
-            raise SystemExit(" [!] The CLI version must be at least v3.3.0-rc1")
```

Comment:
Why not allow `3.3.0-rc1` (and `-rc2`)?

Reply:
The thought is those release candidates will be distant memories soon enough and `v3.3.0` looks cleaner. Easy enough to change back, though.